packages/tocco-ui/src/FormattedValue/util/DateUtils.js

```diff
@@ -1,7 +1,13 @@
-/**
- * Adds the timezone offset of the current environment to the passed date to get the date for the UTC timezone.
- */
-export const convertDateToUTC = date => {
-  const timezoneOffset = new Date().getTimezoneOffset() * 60000
-  return new Date(date.getTime() + timezoneOffset)
+export const parseLocalDate = s => {
```

Comment:
why calling it parse**Local**Date? its it just a parsing of a date in certain format. not 100% sure but i think its the ISO standard format?

I would rename parameter 's' to something more meaningful